addons/binding/org.openhab.binding.onewiregpio/src/main/java/org/openhab/binding/onewiregpio/handler/OneWireGPIOHandler.java

```diff
@@ -102,6 +104,9 @@ public void run() {
                 for (Channel channel : channels) {
                     if (isLinked(channel.getUID().getId())) {
                         publishSensorValue(channel.getUID());
+                    } else {
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
```

Comment:
Seems we are going a bit in circles...
No, a thing is not offline just because it has unlinked channels - this is perfectly fine.
You should remove this else branch again and simply add the setStatus(UNKNOWN) again in line 72, which you originally had there.